projects/demo/src/modules/components/input-chip/examples/1/index.html

```diff
@@ -0,0 +1,9 @@
+<tui-textfield multi>
+    <input
+        placeholder="Placeholder"
```

Comment:
Is it possible to handle anyway the such case ?
<img height="300" alt="demo" src="https://github.com/user-attachments/assets/e78e8135-19b1-4a9a-9961-1b5b17cbfd55" />


Should we add possibility to control the such behavior ?
For example (it does not work now):

```html
<tui-textfield multi>
    <input
        placeholder="Placeholder"
        tuiInputChip
        [style.min-height.px]="50px"
    />
</tui-textfield>

```